src/ua/ithillel/lesson6/main/java/phonebook/PhoneBook.java

```diff
@@ -0,0 +1,24 @@
+package phonebook;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Data
+public class PhoneBook {
+    private List<Record> recordList = new ArrayList<>();
+
+    public void add(Record record) {
+        recordList.add(record);
+    }
+
+    public Record find(String name) {
+        return recordList.stream().filter(el -> el.getName().equals(name)).findFirst().orElse(null);
+    }
+
+    public List<Record> findAll(String name) {
+        return recordList.stream().filter(el -> el.getName().equals(name)).collect(Collectors.toList());
```

Comment:
```suggestion
        return recordList.stream().filter(el -> el.getName().equals(name)).toList(); 
```
лише як альтернатива, до якої сам ще не можу звикнути, з java 16: 
https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/util/stream/Stream.html#toList()